README.md

```diff
@@ -1,20 +1,21 @@
 video-mode-plugin
 =================
 
-A BrightAuthor plugin that changes the video resolution, frame rate, color space and color depth.
+A BrightSign plugin that changes the video resolution, frame rate, color space and color depth.
 
-# Usage
-
-1. Add videomode_plugin.brs to BrightAuthor (File > Presentation Properties > Autorun)
-1. Add a plugin message wherever you want to send it: Advanced tab > Add Command > Send > Send Plugin Message, pick the plugin you added in step 1, and add the text of the the plugin message e.g. "brightsign!videomode!1920x1080x60i!444!10bit" (format below)
+SetSingleScreenVideoMode.brs for non-series 5 single screen players.
```

Comment:
> SetSingleScreenVideoMode.brs for non-series 5 single screen players.

Can SetSingleScreenVideoMode.brs be used for Series 5 single screen players, or Series 5 Multiscreen players while only using single HDMI output? 

Channeling questions customers may directly / indirectly ask or find confusion with when determining "which plugin to use".

Similar set of questions for...
> SetMultiScreensVideoMode.brs for series 5 multi screens players.

Can use on series 5 multi screen with only 1 hdmi output configured?

Reply:
> Can SetSingleScreenVideoMode.brs be used for Series 5 single screen players, or Series 5 Multiscreen players while only using single HDMI output?
No, SetSingleScreenVideoMode returns a string, and series 5 players call setVideoMode(as Array) which requires a parameter to be a screen Array. I have some related info added to the plugin. 

> Can use on series 5 multi screen with only 1 hdmi output configured?
Yes, it is possible if you just configure 1 HDMI, e.g. modifying screenArray[0]

Reply:
With the above understanding, could we update the below note to indicate single or multi screen _series 5_ hardware is unimportant here, what is important is that the hardware is series 5. 

> SetMultiScreensVideoMode.brs for series 5 multi screens players.

SetMultiScreensVideoMode.brs is used for series 5 players. SetScreenModes is used by series 5 hardware to support 1 to many connected screens.

Reply:
I want to set a flag here with the above comment that this can be a potential break point for series5 single screen players when bacon and the plugin are using different data structure.

Reply:
Sounds good, thanks for the note. Once there are new players with new videoMode instructions, we can update this GitHub page along the way.